### DISCUSSION TITLE
Device Id Pre-Text - Reference list

### DISCUSSION THREAD

**Discussion by twister36** [ESPresense/ESPresense#969] [General]
Do we have a reference list of what the pre-text in a device id equals?

For example:
iBeacon:{id} = iBeacon app such as Home Assistant Companion App
tile:{id} = Title tag trackers
msft:{id} = Microsoft (in my case surface laptop)

I see many other ids that I have no clue what they are, and knowing the pre-text could help to identify them.

With that in mind, do we have a master list of these pre-texts in the ids and what they reference?

**Comment by DTTerastar** [comment]
samsung: = manuf data w/ 0075 
mifit: = manuf data w/ 0175
sonos: = manuf data w/ 06a7
garmin: = manuf data w/ 4d4b

https://github.com/ESPresense/ESPresense/blob/d6107ab14f8061bd41490f2ddb253616647c23bd/lib/BleFingerprint/BleFingerprint.cpp search for setId and you can see all of them
